native/src/routes/offers/containers/OffersContainer.js

```diff
@@ -61,11 +61,9 @@ const OffersContainer = ({ theme, t, navigation, route }: OffersPropsType) => {
 
       if (isExternalUrl && postData) {
         // HTTP POST is neither supported by the InAppBrowser nor by Linking, therefore we have to open it in a webview
-        if (postData) {
-          navigation.push(EXTERNAL_OFFER_ROUTE, { url: path, shareUrl: path, postData })
-        } else {
-          openExternalUrl(path)
-        }
+        navigation.push(EXTERNAL_OFFER_ROUTE, { url: path, shareUrl: path, postData })
+      } else if (isExternalUrl) {
+        openExternalUrl(path)
```

Comment:
:facepalm: Thanks! Could you add a test for this please?

Reply:
> I tried adding tests for the navigation behavior here. But as Offers are mocked manually I didn't find a clean solution. Does anyone know a way to cleanly test this here?

Ideas?